store/event-store/src/test/java/org/calrissian/accumulorecipes/eventstore/impl/AccumuloEventStoreTest.java

```diff
@@ -136,6 +136,8 @@ public void testVisibility() {
         assertEquals(1, Iterables.get(actualEvent1, 1).size());
     }
 
+
+    @Ignore // ignoring for now. We may not need tuples to expire
```

Comment:
Same question as above.  If we are removing the capability for tuples to expire, then there should be a ticket created and linked to the pull request that documents why this is being removed as a feature.